docs/src/main/asciidoc/se/scheduling.adoc

```diff
@@ -112,7 +112,7 @@ include::{sourcedir}/se/SchedulingSnippets.java[tag=snippet_4, indent=0]
 For simple fixed rate invocation use .
 
 [source,java]
-.Example of scheduling with fixed rate use `Scheduling.fixedRate()` builder.
+.Example of scheduling with fixed rate use `FixedRate.builder()` builder.
```

Comment:
"using"?

Reply:
That is original text that I did not re-read. I will fix it.